keycloak-init-container/extensions.sh

```diff
@@ -1,52 +1,63 @@
-#!/bin/bash -e
+#!/bin/bash
 
-EXTENSIONS_VOLUME="${EXTENSIONS_VOLUME:-/opt/extensions}"
+shopt -so errexit pipefail
```

Comment:
@apeschel 

In which way the meaning of the previous `shopt` commad differs from `set -xeuo pipefail`? Did `shellcheck` suggest this form for you?

Or if the display of expanded `PS4` value isn't desired, could `set -euo pipefail` form be used? The latter form looks a more commonly used one for me, but that's just a nuance.

Reply:
@iankko `shopt` is a personal preference/style thing - I just find shopt has a cleaner interface than set.

I'll switch it to `set` since that appears to be the local preference.

Reply:
@iankko I'm curious where the `set -xeuo pipefail` comes from though -- there was just a `-e` on the shebang line previously.

Reply:
> @iankko I'm curious where the `set -xeuo pipefail` comes from though -- there was just a `-e` on the shebang line previously.

From other scripts in this repo, e.g. here:
https://github.com/keycloak/keycloak-containers/blob/master/server/tools/docker-entrypoint.sh#L2

The `-x` there is just to print the expanded form of the statements to be executed. Can be removed if not desired.